### ISSUE TITLE
Rename binary: jq -> jq.node

### ISSUE THREAD

**Issue by chocolateboy** [FGRibreau/jq.node#1]
Calling the binary `jq` obviously conflicts with [`jq`](https://github.com/stedolan/jq), and none of the installers (`npm`, `ied`, `yarn`) support renaming binaries. 

I assumed that the binary would be called `jq.node` and that giving it a shorter/friendlier name would be left up to the user e.g. with an alias:

``` sh
alias jqn="jq.node"
```

The documentation mentions several cases where `jq` is better than `jq.node` so it seems perverse for a complementary tool to clobber it without the user's consent.

**Comment by FGRibreau** [comment]
> none of the installers (npm, ied, yarn) support renaming binaries

Why not:

```
postinstall:"(JQ=$(which jq) && DIR=$(dirname $JQ) && mv $JQ $DIR/jq.old) || true"
```

?

I prefer to use jq.node as my default (if I install it) and then still have access to the old jq for special tasks (e.g. handling very large file)

Clearly, this is open for debate :)

**Comment by chocolateboy** [comment]
> I prefer to use jq.node as my default (if I install it)

``` sh
alias jqc=$(which jq)
alias jq=$(which jq.node)
```

> Why not:
> 
> ```
>  postinstall:"(JQ=$(which jq) && DIR=$(dirname $JQ) && mv $JQ $DIR/jq.old) || true"
> ```
> 
> ?
- It doesn't work on Windows
- It doesn't work with `/usr/bin/jq` &c. (without root)
- It may not even work with `$HOME/bin/jq` &c.
- Moving/renaming a binary on my system is the last thing I'd expect this (or any other piece of software) to do

**Comment by FGRibreau** [comment]
> - It doesn't work on Windows
> - It doesn't work with /usr/bin/jq &c. (without root)
> - It may not even work with $HOME/bin/jq &c.

Definitely had that in mind when I suggested the postinstall. It will only work with user-privilege and that's why the "true" is at the end.

About the windows part, I'm not even sure that piping will work the right way when I see projects like this https://github.com/cowboy/node-exit ...

I took for granted that jq only worked on *nix platform. But indeed it work on windows (my bad, thanks for bringing that up!) however I'm not sure that I want to support windows as a plateform for jq.node and prefer to focus on *nix platforms.
